vochain/transaction.go

```diff
@@ -332,37 +332,37 @@ func (app *BaseApplication) AddTx(vtx *VochainTx, commit bool) (*AddTxResponse,
 // VoteEnvelopeCheck is an abstraction of ABCI checkTx for submitting a vote
 // All hexadecimal strings should be already sanitized (without 0x)
 func (app *BaseApplication) VoteEnvelopeCheck(ve *models.VoteEnvelope, txBytes, signature []byte,
-	txID [32]byte, forCommit bool) (*models.Vote, error) {
-
+	txID [32]byte, forCommit bool) (*models.Vote, []byte, error) {
+	voterID := make([]byte, 0)
```

Comment:
This is a bit weird. IMO this method should return just `nil` and voterID only when it has a valid value.